src/Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs

```diff
@@ -351,7 +351,7 @@ public SwitchParameter BreakAll
         }
```

Comment:
@KirkMunro, your last commit had 2 failures in `PowerShell-CI-windows`
Basic debugger tests.The value of $? should be preserved when exiting the debugger.Should show 2 debugger commands were invoked
```powershell
Expected 2, but got 3.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\Debugging\Debugging.Tests.ps1: line 41
41:             $results.Count | Should -Be 2
```
Basic debugger tests.The value of $? should be preserved when exiting the debugger.Should have $false output from the first $? command
```powershell
Expected $false, but got $true.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\Debugging\Debugging.Tests.ps1: line 46
46:             $results[0].Output | Should -Not -BeTrue
```